cpp/include/cudf/strings/strings_column_view.hpp

```diff
@@ -107,19 +107,21 @@ class strings_column_view : private column_view {
    * @brief Returns the internal column of chars
    *
    * @throw cudf::logic error if this is an empty column
+   * @param stream CUDA stream used for device memory operations and kernel launches
    * @return The chars column
    */
-  [[nodiscard]] column_view chars() const;
+  [[nodiscard]] column_view chars(rmm::cuda_stream_view stream) const;
```

Comment:
This member should be removed. Did you mean to add a stream parameter to `chars_begin()`?